docs/reference/cluster-config.txt

```diff
@@ -848,6 +848,43 @@ The master supports the following configuration settings:
                signed by a well-known CA; cannot be specified if
                ``skip_verify`` is enabled.
 
+-  ``scim``: (EE-only) Specifies whether the SCIM service is enabled and
```

Comment:
I don't remember talking about pulling the docs in, but it makes a lot of sense, considering it's probably have the way anyone would figure out about EE.

Reply:
I've cleared this with Angela, we're going to have unified docs.

Reply:
sweet.